src/backend/app/waypoints/waypoint_routes.py

```diff
@@ -105,16 +105,19 @@ async def get_task_waypoint(
 
     if project.is_terrain_follow:
         dem_path = f"/tmp/{uuid.uuid4()}/dem.tif"
-        get_file_from_bucket(
-            settings.S3_BUCKET_NAME, f"projects/{project_id}/dem.tif", dem_path
-        )
+        try:
+            get_file_from_bucket(
```

Comment:
As you rightly say below, once everything is working and we are happy, this definitely needs to be changed getting elevation data by querying the cloud optimised DEM. A DEM can be quite large, so downloading every time we process imagery is very inefficient